### ISSUE TITLE
Edge Chromium App mode

### ISSUE THREAD

**Issue by Dtar380** [python-eel/Eel#718]
I've seen this issue already [#251](https://github.com/python-eel/Eel/issues/251) which is from 2020, and Im asking my self, ¿Will you add that feature to the versions to come? Because it will be pretty handy to be able to use Edge in App mode with a fresh install of the package.

**Comment by Meenapintu** [comment]
Looks like this is a dead project. There are so many open pull request. 

I was also looking for it. Then I forked it and created a code change to support app mode in edge.  

I created a pull request and waiting for it's merge https://github.com/python-eel/Eel/pull/744 , however,  until this get merged , you can use my forked repo. https://github.com/yehigo/Eel

**Comment by ChrisKnott** [comment]
@Meenapintu Yes unfortunately none of the previous maintainers have capacity to do so any more. Do you want to maintain the main repo?

**Comment by thatfloflo** [comment]
@ChrisKnott just to throw my head in the ring here, I'd be happy to contribute to contribute toward maintaining eel if that helps.
